htdocs/install/mysql/tables/llx_knowledgemanagement_knowledgerecord.sql

```diff
@@ -21,6 +21,7 @@ CREATE TABLE llx_knowledgemanagement_knowledgerecord(
 	date_creation datetime NOT NULL, 
 	tms timestamp, 
 	last_main_doc varchar(255), 
+	default_lang varchar(6),
```

Comment:
Rename the field "lang" instead of "default_lang" because it is the language of article not a default that can be overwrite.